### PR TITLE
chore: Bump Kotlin plugin version to 1.4

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Comment by KazuCocoa** [comment]
👣 (Just for marking this pr)

File: espresso-server/app/build.gradle.kts [hunk 1]
```diff
@@ -99,7 +99,7 @@ dependencies {
     androidTestImplementation("androidx.test:rules:${Version.testlib}")
     androidTestImplementation("com.google.code.gson:gson:${Version.gson}")
     androidTestImplementation("org.nanohttpd:nanohttpd-webserver:${Version.nanohttpd}")
-    androidTestImplementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Version.kotlin}")
+    androidTestImplementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:${Version.kotlin}")
```
Comment (KazuCocoa) [inline espresso-server/app/build.gradle.kts:102]: Tried to use `implementation`, with JDK 11, with kotlin 1.4.31 etc, but still no luck.
```
[debug] [Espresso] [Instrumentation] io.appium.espressoserver.EspressoServerRunnerTest:
[debug] [Espresso] [Instrumentation] Error in startEspressoServer(io.appium.espressoserver.EspressoServerRunnerTest):
[debug] [Espresso] java.lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/internal/Intrinsics;
[debug] [Espresso] 	at io.appium.espressoserver.lib.helpers.AndroidLogger.info(Unknown Source:2)
[debug] [Espresso] 	at io.appium.espressoserver.lib.http.Server.stop(Server.kt:103)
[debug] [Espresso] 	at io.appium.espressoserver.EspressoServerRunnerTest.startEspressoServer(EspressoServerRunnerTest.kt:51)
[debug] [Espresso] 	at java.lang.reflect.Method.invoke(Native Method)
```

Comment (mykola-mokhnach) [reply espresso-server/app/build.gradle.kts:102]: this issue has been resolved

File: espresso-server/gradle.properties [hunk 2]
```diff
@@ -22,3 +22,5 @@ android.useAndroidX=true
 
 org.gradle.vfs.watch=true
 org.gradle.caching=true
+# https://youtrack.jetbrains.com/issue/KT-46708
+kotlin.stdlib.default.dependency=false
```
Comment (KazuCocoa) [inline espresso-server/gradle.properties:26]: oh, great...

**Review by KazuCocoa — APPROVED** [review]

**Review by jlipps — APPROVED** [review]

**Pull request merged**